Build failure with Python 3.10

If running with Python 3.10 then the tests don't pass.

```bash
[...]
self.name)
                        )
E               AttributeError: Command: no attribute
'GW_LEAVE_LEARN_STATE_REQ'

/usr/lib64/python3.10/enum.py:146: AttributeError
=========================== short test summary info
============================
FAILED
test/frame_leave_learn_state_req_test.py::TestFrameLeaveLearnStateRequest::test_frame_from_raw
======================== 1 failed, 287 passed in 0.84s
=========================
```

See https://bugzilla.redhat.com/show_bug.cgi?id=1928062 for details.

Tanks

--- a/pyvlx/api/frame_creation.py
+++ b/pyvlx/api/frame_creation.py
@@ -177,7 +177,7 @@ def create_frame(command):
         return FrameNodeStatePositionChangedNotification()
     if command == Command.GW_LEAVE_LEARN_STATE_CFM:
         return FrameLeaveLearnStateConfirmation()
-    if command == command.GW_LEAVE_LEARN_STATE_REQ:
+    if command == Command.GW_LEAVE_LEARN_STATE_REQ:
         return FrameLeaveLearnStateRequest()
 
     return None

